release_notes/Service_Fabric_ReleaseNotes_80.md

```diff
@@ -0,0 +1,113 @@
+# Microsoft Azure Service Fabric 8.0  Release Notes
+
+This release includes the bug fixes and features described in this document. This release includes runtime, SDKs and Windows Server Standalone deployments to run on-premises. 
+
+The following packages and versions are part of this release:
+
+| Service | Platform | Version |
+|---------|----------|---------|
+|Service Fabric Runtime| Ubuntu 16 <br> Ubuntu 18 <br> Windows | 8.0.511.1 <br>  8.0.511.1804 <br> 8.0.512.9590  |
+|Service Fabric for Windows Server|Service Fabric Standalone Installer Package | 8.0.512.9590 |
+|.NET SDK |Windows .NET SDK <br> Microsoft.ServiceFabric <br> Reliable Services and Reliable Actors <br> ASP.NET Core Service Fabric integration| 8.0.512  <br> 8.0.512 <br> 8.0.512 <br> 8.0.512 |
+|Java SDK  |Java for Linux SDK  | 1.0.5 |
+|Service Fabric PowerShell and CLI | AzureRM PowerShell Module  <br> SFCTL |  0.3.15  <br> 8.0.0 |
+
+
+## Contents 
+
+Microsoft Azure Service Fabric 8.0 Release Notes
+
+* [Key Announcements](#key-announcements)
+* [Current And Upcoming Breaking Changes](#Current-And-Upcoming-Breaking-Changes)
+* [Service Fabric Runtime](#service-fabric-runtime)
+* [Service Fabric Common Bug Fixes](#service-fabric-common-bug-fixes)
+* [Repositories and Download Links](#repositories-and-download-links)
+
+
+## Key Annoucements
```

Comment:
We should make sure we call out .Net 5 support as GA as one of the top items.

Reply:
Done.